Adiciona componente de listagem: tabela reutilizável

darlanbarroso opened the pull request.

darlanbarroso committed:
Adiciona componente de listagem: tabela reutilizável

gustavo-reinaldo reviewed (changes requested):
Separação de arquivos de acordo com a estrutura do projeto.

darlanbarroso committed:
refactor: modularizar componente de tabela reutilizável (HTML, CSS e JS)

- Separado components/tabela-reutilizavel.html da estrutura completa para conter apenas o HTML
- Criado arquivo css/components/tabela-reutilizavel.css com todos os estilos do componente
